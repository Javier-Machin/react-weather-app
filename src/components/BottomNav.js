import React from 'react';

// Text and links displayed at the bottom of the app

const BottomNav = () => {
  return (
    <footer className="bottom-nav">
      <a href="http://www.javiermachin.com">© Javier Machín</a>
      <a
        className="vector-credit"
        href="https://www.freepik.com/free-photos-vectors/house"
      >
        House vector created by Rawpixel.com - Freepik.com
      </a>
    </footer>
  );
};

export default BottomNav;
