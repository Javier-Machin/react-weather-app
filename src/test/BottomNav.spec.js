import React from 'react';
import { shallow } from 'enzyme';
import BottomNav from '../components/BottomNav';

describe('BottomNav', () => {
  it('should render correctly', () => {
    const component = shallow(<BottomNav />);

    expect(component).toMatchSnapshot();
  });
});
