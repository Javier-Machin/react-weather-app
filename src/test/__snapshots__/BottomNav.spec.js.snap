// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomNav should render correctly 1`] = `
<footer
  className="bottom-nav"
>
  <a
    href="http://www.javiermachin.com"
  >
    © Javier Machín
  </a>
  <a
    className="vector-credit"
    href="https://www.freepik.com/free-photos-vectors/house"
  >
    House vector created by Rawpixel.com - Freepik.com
  </a>
</footer>
`;
